build.rs

```diff
@@ -357,10 +357,19 @@ fn blob_backends(conf: &Config, build_dir: &std::path::PathBuf) -> (Vec<String>,
     if conf.use_lib {
         let afpath  = match env::var("AF_PATH") {
             Ok(af_path) => PathBuf::from(&af_path),
-            Err(_)      => panic!("Error use_lib is defined, but AF_PATH is not defined"),
+            Err(_)      => {
+                println!("WARNING! USE_LIB IS DEFINED, BUT AF_PATH IS NOT DEFINED, TRYING TO FIND LIBRARIES FROM KNOWN DEFAULT LOCATIONS");
+                PathBuf::from("/usr/")
+            },
         };
         let libpath = afpath.join("lib");
```

Comment:
No, there won't be error. I think we can `panic` or use some default paths. We are not checking for existence of the libraries, we are only adding paths like `-L` we use for gcc/g++ link options. If required library is not present in any of search paths, it will throw linking errors.